content/blog/2023-05-02-response-to-bitcoin-magazine-article-about-tokenization/index.md

```diff
@@ -0,0 +1,124 @@
+
+That people are now experimenting with the technology, playing with NFTs, speculating, scamming, and losing money is just the prelude and the training wheels phase of the system. All these activities are only serving as a learning process and showing the abilities of the technology. 
+
+### Mr. Koss Wrote:
+
+*“Furthermore, a “blockchain” is simply a ledger, or record of who owns what. With Ethereum especially, there is no meaningful link to the real world which would allow for native contract enforcement, preventing rug pulls of these token holders. The whole system ultimately relies upon legacy law enforcement and the judicial system to uphold the property rights of these investors — a system that appears to be increasingly hostile in enforcement actions against the crypto industry writ large.”*
+
+### Our response:
+
+Exactly! A programmable blockchain is a ledger with a record of who owns what, but in the case of ETC (not Ethereum), incredibly secure. 
+
+There is no meaningful link between current systems and physical assets either! We would argue the current systems are even less linked than if assets were represented on the blockchain.
+
+Yes, the whole system will always be dependent on laws and their enforcement. This will never be delinked even for onchain assets as BTC and ETC. You may have 1 BTC or 100 ETC in the account you control through your private key, but that does not mean that the asset is your property. Legal property is not the same as physical access and control. The latter is what blockchains guarantee, not the former. 
+
+The law will eventually adapt and futile resistance will wither.
+
+### Mr. Koss Wrote:
+
+*“Bitcoin adoption is fundamentally different, a fact that crypto folks seem to misunderstand completely. Rather than mindlessly tokenizing assets, Bitcoin seeks to fix the monetary issues that drive a desire to do so in the first place. By serving as an actual store of value, bitcoin will drain the monetary premium that real estate has accrued over the past decades due to the broken monetary system. Under a bitcoin standard, housing will ultimately collapse to its utility value, making houses affordable once more to the everyday citizen.”*
+
+### Our response:
+
+Mindlessly tokenizing any assets for short term speculation is silly, but the real benefits of transferring and legally recognizing property records on truly secure blockchains will be great for society. The same guarantees that BTC and ETC bring to their native currencies will be enjoyed by these property systems. 
+
+That Bitcoin and Ethereum Classic will be used as sound money and as stores of value will definitely drain the premiums of all other assets, but valuation is a different concept than property registration and transfer. Proof of work blockchains provide a new set guarantees for property in general, including real estate.
```

Comment:
Line 102: Verb agreement;  Add preposition
```suggestion
That Bitcoin and Ethereum Classic will be used as sound money and as stores of value will definitely drain the premiums of all other assets, but valuation is a different concept than property registration and transfer. Proof of work blockchains provides a new set of guarantees for property in general, including real estate.
```

Reply:
There is a typo in the suggestion "provides" is "provide". I will fix the second typo adding "of" from the branch.